scripts/create_sample_worksheet.py

```diff
@@ -20,16 +20,17 @@ class SampleWorksheet:
     _WORKSHEET_FILE_PATH = '/tmp/sample-worksheet-temp.txt'
```

Comment:
I also see this change in another PR.

Reply:
It's because this PR is stacked on that one. I'm opening a PR per commit.